app/frontend/src/features/communities/CommunityPage/EventsSection.tsx

```diff
@@ -1,27 +1,17 @@
-import Button from "components/Button";
 import HorizontalScroller from "components/HorizontalScroller";
 import { CalendarIcon } from "components/Icons";
 import TextBody from "components/TextBody";
 import {
   EVENTS_EMPTY_STATE,
   EVENTS_TITLE,
-  SEE_MORE_EVENTS_LABEL,
 } from "features/communities/constants";
-import { Community } from "proto/communities_pb";
-import { useHistory } from "react-router-dom";
-import { routeToCommunity } from "routes";
 
 import { useCommunityPageStyles } from "./CommunityPage";
 import EventCard from "./EventCard";
 import SectionTitle from "./SectionTitle";
 
-export default function EventsSection({
-  community,
-}: {
-  community: Community.AsObject;
-}) {
+export default function EventsSection() {
```

Comment:
Is this being used anywhere at the moment once you removed from `CommunityPage`? If not, might be better to leave this one to avoid merge conflicts as I'm making changes to this file soon to hook in real event data 😅